main/realmain.cc

```diff
@@ -618,8 +628,6 @@ int realmain(int argc, char *argv[]) {
 
         if (!opts.storeState.empty()) {
             gs->markAsPayload();
-            // Store file hashes for LSP.
-            hashing::Hashing::computeFileHashes(gs->getFiles(), *logger, *workers);
```

Comment:
Replaced with logic above, which indexes and hashes at the same time. This should speed up payload generation. :)